### DISCUSSION TITLE
fan can't go under 2800 even I put curve to boost 0

### DISCUSSION THREAD

**Discussion by xli659** [T-Troll/alienfx-tools#437] [General]
When doing the manually fan control, the lowest fan will stay around 2800 RPM (48% fan speed), even when I put the curve boost to zero. It only can control fan speed above 2800 RPM. Is this expected behavior? I am on M16 R2. Is this model supported?

**Comment by Wromper** [comment]
The same issue happens with me, the starting speed seems to be about 2000 rpm, which may be correct.  However, I have another RPM related issue.  Even with the boost a flat 0, the fans will spin up to over 3000 RPM when temps are too low, even around 70C.  How can I force the fans to never exceed a certain RPM, regardless of temp?
